packages/wallets/src/features/cashier/modules/WithdrawalVerification/WithdrawalVerificationSent/WithdrawalVerificationSent.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
-import EmailSent from '../../../../../public/images/email-sent.svg';
+import { WalletButton } from '../../../../../components';
```

Comment:
Can we use alias?

Reply:
Not yet implemented @balakrishna-deriv. We discussed to use alias after we move to a new repo. If we move 👀

Reply:
Currently we cant 😬 , it will conflicts with other packages